Move preview APIs article to runtime libraries section of docs (#44436)
dotnet/docs: docs/fundamentals/runtime-libraries/preview-apis.md

---
title: Preview APIs
description: Learn how to mark .NET APIs as preview.

> ms.date: 4/4/2024

Reviewer: Should this be a more current date?

> ms.date: 4/4/2024

Author: ```suggestion
ms.date: 01/27/2025
```

---

# Preview APIs

The .NET platform takes compatibility seriously. As such, the library ecosystem tends to avoid making breaking changes, especially with respect to the API.

Nonetheless, when designing APIs, it's important to be able to collect feedback from users and make changes to the API based on that feedback if necessary. To avoid surprises, you should understand which APIs you use are considered stable and which ones are still in active development and might change.

> There are multiple ways an API can express to be in preview form:

Reviewer: Hmm...  

```suggestion
There are multiple ways an API can express its in preview form:
```
